<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_CriarMatrizA">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>Criar Matriz A</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_LinA">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(71, 89, 87);
color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ColA">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(71, 89, 87);
color: white;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Resultado">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(71, 89, 87);
color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>    Matriz A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
</string>
    </property>
    <property name="text">
     <string>Qtd Linhas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
</string>
    </property>
    <property name="text">
     <string>Qtd Colunas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ColB">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(71, 89, 87);
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58);
color:yellow;
</string>
    </property>
    <property name="text">
     <string>Qtd Colunas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58);
color:yellow;
</string>
    </property>
    <property name="text">
     <string>Qtd Linhas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_LinB">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(71, 89, 87);
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>747</x>
      <y>19</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58);
color:yellow;
font-size: 16px;
border-radius: 8px;
</string>
    </property>
    <property name="text">
     <string>   Matriz B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CriarMatrizB">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58);
border:2px solid yellow;
color:yellow;
border-radius: 8px;
</string>
    </property>
    <property name="text">
     <string>Criar Matriz B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Sup_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58) ;
color:  yellow;
border-radius: 8px;
border: 1px solid yellow;</string>
    </property>
    <property name="text">
     <string>▲SUP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_potencia_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58) ;
color:  yellow;
border-radius: 8px;
border: 1px solid yellow;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Id_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58) ;
color:  yellow;
border-radius: 8px;
border: 1px solid yellow;</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Simetrica_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58) ;
color:  yellow;
border-radius: 8px;
border: 1px solid yellow;</string>
    </property>
    <property name="text">
     <string>Simetrica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MultiK_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58) ;
color:  yellow;
border-radius: 8px;
border: 1px solid yellow;</string>
    </property>
    <property name="text">
     <string>* (K)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_trans_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58) ;
color:  yellow;
border-radius: 8px;
border: 1px solid yellow;</string>
    </property>
    <property name="text">
     <string>Trans</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Inf_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58) ;
color:  yellow;
border-radius: 8px;
border: 1px solid yellow;</string>
    </property>
    <property name="text">
     <string>▲ INF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(58, 58, 58);
border:2px solid yellow;
color:yellow;
border-radius: 8px;
</string>
    </property>
    <property name="text">
     <string>        MATRIZ B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Soma">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
border-radius: 15px;
color: yellow;
border: 2px solid rgb(255, 230, 90);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Multi">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
border-radius: 15px;
color: yellow;
border: 2px solid rgb(255, 230, 90);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Sub">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
border-radius: 15px;
color: yellow;
border: 2px solid rgb(255, 230, 90);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Igual">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
border-radius: 15px;
color: yellow;
border: 2px solid rgb(255, 230, 90);</string>
    </property>
    <property name="text">
     <string>==</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dif">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
border-radius: 15px;
color: yellow;
border: 2px solid rgb(255, 230, 90);</string>
    </property>
    <property name="text">
     <string>!=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>        MATRIZ A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_potencia">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_MultiK">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>* (K)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Id">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Sup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>▲SUP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Simetrica">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>Simetrica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_trans">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>Trans</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Inf">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 0) ;
color:  black;
border-radius: 8px;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>▲ INF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
